backend/question/src/services/get/types.ts

```diff
@@ -0,0 +1,75 @@
+import type { IServiceResponse } from '@/types';
+
+//=============================================================================
+// /get
+//=============================================================================
+export type IGetQuestionsPayload = {
+  // Filters
+  questionName?: string;
+  difficulty?: string;
+  topic?: string;
+  // Pagination
```

Comment:
In the sample data given, some questions have multiple topics, so this might have to be an array

Reply:
fixed